src/tls13.c

```diff
@@ -4630,11 +4630,19 @@ int DoTls13ClientHello(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
         if (!ssl->options.downgrade) {
             WOLFSSL_MSG("Client trying to connect with lesser version than "
                         "TLS v1.3");
+#ifdef OPENSSL_EXTRA
```

Comment:
would this be something that could also fall under the macro WOLFSSL_EXTRA_ALERTS?

Reply:
Added new alerts to `WOLFSSL_EXTRA_ALERTS`.